1_basics.ipynb

```diff
@@ -143,23 +256,75 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "1.  Start the simulator environment by entering \"docker-compose up\" command.\n",
+    "\n",
+    "\n",
+    "2.  Access http://localhost:3000/ from your browser and open the simulator screen. The screen of Gazebo is displayed.\n",
```

Comment:
"2.  Access http://localhost:3000/ **using** your browser and open the simulator screen. The screen **for** Gazebo **will be** displayed.\n",